radio/src/translations/de.h

```diff
@@ -1152,8 +1152,8 @@
 #define TR_MENU_INVERT                 "Invertieren<!>"
 #define TR_JITTER_FILTER               "ADC Filter"
 #define TR_DEAD_ZONE                   "Dead zone"
-#define TR_RTC_CHECK                   TR("Check RTC", "Check RTC voltage")
-#define TR_AUTH_FAILURE                "Auth-failure"
+#define TR_RTC_CHECK                   TR("Prüfe RTC", "Prüfe RTC Spannung")
```

Comment:
If this is a warning to prompt the user to check the RTC battery, this sounds like it is currently checking the voltage.
"RTC Prüfen" and "RTC Spannung prüfen"?

Reply:
done

Reply:
with "RTC Spannung prüfen" there is a line break.
![image](https://user-images.githubusercontent.com/15068446/192710584-27d04dc6-839c-4fce-b101-0ab62d802e32.png)

Should I then call it "RTC Volt prüfen"?

Reply:
![image](https://user-images.githubusercontent.com/15068446/192718793-fbc39109-1334-41d1-80e5-06124beb24cf.png)

Reply:
new plan: "Akku Spannungsbereich" is also much too long. So I've shorten it with the same abbreviation like "RTC Spannung prüfen". I think it looks good:
![image](https://user-images.githubusercontent.com/15068446/192724479-ae0ab381-8e36-4d7e-8402-835f56bb2daa.png)